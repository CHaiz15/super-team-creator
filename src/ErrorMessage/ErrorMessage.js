import React from 'react'
import './ErrorMessage.scss'
import { NavLink } from 'react-router-dom'

export const ErrorMessage = () => {
  return (
    <>
      <h1 className='creation-title'>Error</h1>
      <section className='error-container'>
        <p className='error-box'>One of the heros/villians you entered does not exist try again.</p>
        <NavLink className='build-btn' to='/create'>Okay</NavLink>
      </section>
    </>
  )
}

export default ErrorMessage