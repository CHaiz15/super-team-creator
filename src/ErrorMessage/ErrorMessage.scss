.error-container {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin-top: 15%;
}

.error-box {
  font-size: 1.2em;
  height: 4rem;
  background-color: white;
  border-radius: 10px;
  width: 30rem;
  display: flex;
  flex-direction: column;
  align-items: center;
  opacity: 0.95;
  animation: slide-top 0.5s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}

@keyframes slide-top {
  0% {
    transform: translateY(0);
  }
  100% {
    transform: translateY(-100px);
  }
}