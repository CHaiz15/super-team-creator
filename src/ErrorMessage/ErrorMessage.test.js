import React from 'react';
import { ErrorMessage } from './ErrorMessage.js';
import { shallow } from 'enzyme'

describe('ErrorMessage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ErrorMessage/>);
  })  
  it('should be an instance of the ErrorMessage component', () => {
      expect(wrapper).toMatchSnapshot();
  });
})
