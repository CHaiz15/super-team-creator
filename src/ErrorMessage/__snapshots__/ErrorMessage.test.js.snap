// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should be an instance of the ErrorMessage component 1`] = `ShallowWrapper {}`;
